docs/website-metadata.md

```diff
@@ -105,6 +112,128 @@ references
+
+# Display the first 10 pages as JSON (de-indexed)
+JSON({f"page_{i+1}": page for i, page in enumerate(pages[:10])})
+```
+
+### Get all instances of a specific content type
+
+You can filter references by their `kind` to find all figures, tables, citations, or other content types:
+
+```{code-cell} python
+from collections import Counter
+
+# Count all content types
+all_types = Counter(ref["kind"] for ref in xref_data["references"])
+
+# Show examples of each type
+figures = [ref for ref in xref_data["references"] if ref["kind"] == "figure"][:3]
+tables = [ref for ref in xref_data["references"] if ref["kind"] == "table"][:3]
+internal_refs = [ref for ref in xref_data["references"] if ref.get("implicit")][:3]
+external_refs = [ref for ref in xref_data["references"] if ref["kind"] == "heading" and not ref.get("implicit")][:3]
+
+JSON({
+    "all_content_types": dict(all_types),
+    **{f"figure_{i+1}": fig for i, fig in enumerate(figures)},
+    **{f"table_{i+1}": tbl for i, tbl in enumerate(tables)},
+    **{f"internal_ref_{i+1}": ref for i, ref in enumerate(internal_refs)},
+    **{f"external_ref_{i+1}": ref for i, ref in enumerate(external_refs)}
+})
+```
+
+### Access page metadata and source information
+
+Each reference includes a `data` field pointing to a JSON file with complete metadata:
+
+```{code-cell} python
+# Get first page and fetch its metadata
+page = next(ref for ref in xref_data["references"] if ref["kind"] == "page")
+data_url = site_url + page["data"]
+
+print(f"Retrieving URL: {data_url}")
+page_data = requests.get(data_url).json()
+
+JSON(page_data["frontmatter"])
+```
+
+### Download the MyST AST of a page
+
+The JSON file at each page's `data` URL contains the complete MyST Abstract Syntax Tree (AST):
+
+```{code-cell} python
+# Get the index page and fetch its AST
+index_page = next(ref for ref in xref_data["references"] if ref["url"] == "/")
+data_url = site_url + index_page["data"]
+
+print(f"Retrieving URL: {data_url}")
+myst_ast = requests.get(data_url).json()
+
+JSON({
+    "kind": myst_ast["kind"],
+    "slug": myst_ast["slug"],
+    "mdast_children_count": len(myst_ast["mdast"]["children"]),
+    "mdast_first_child": myst_ast["mdast"]["children"][0]
+})
+```
+
+### Find and download the exports and source file of a page
+
+You can locate the original source file and available exports for each page using the page's JSON data:
+
+```{code-cell} python
+# Get a page with "quickstart" in the URL
+example_page = next(ref for ref in xref_data["references"] if "quickstart" in ref["url"])
+data_url = site_url + example_page["data"]
+
+print(f"Retrieving URL: {data_url}")
+page_data = requests.get(data_url).json()
+
+JSON({
+    "page_url": example_page["url"],
+    "page_title": page_data["frontmatter"]["title"],
+    "source_file": page_data["location"],
+    "exports": page_data["frontmatter"]["exports"]
+})
+```
+
+Download and display the source file content from the CDN:
```

Comment:
Unclear why we're now dealing with a CDN. Will probably be clear once I see the cell output.